general-information.md

```diff
@@ -283,6 +283,8 @@ Vehicles can enter the `unknown` state to and from any other state with the foll
 | `unknown` | `on_trip`   | `comms_restored`        | The vehicle transmitted status information after a period of being out of communication. |
 | `unknown` | `on_trip`   | `unspecified`        | The provider cannot definitively state how a vehicle started a trip. |
 | `on_trip` | `elsewhere`   | `trip_leave_jurisdiction` | A vehicle on a trip left the jurisdiction |
+| `on_trip` | `on_trip `   | `trip_enter_geography` | The vehicle enters one or more Geographies managed by Policy during an active trip. |
+| `on_trip` | `on_trip `   | `trip_leave_geography` | The vehicle leaves one or more Geographies managed by Policy during an active trip. |
```

Comment:
It seems like we need to define an event type in order to effect a status event for crossing geometry boundaries, right?

But I see your point about how these new events I've added are different because they represent internal transitions. What do you think about having a separate table of internal transition events? I'm imagining there may be other possible useful internal events we might some day consider:
- a stationary vehicle updates its location due to acquisition of a better GPS fix
- a parked vehicle is picked up and carried to a new location
- a parked vehicle is charged
- an in motion vehicle experiences a possible crash event

Reply:
What I propose is that we make Policy responsive to telemetry as well as events.  We need that anyway to enforce `speed` rules.  Let me put together a specific (hypothetical) example that expresses your "any trip that intersects downtown" use-case.

Reply:
I'm unclear on how that would work for MDS Provider

Reply:
See below, it would require using Trip data.

Reply:
Ah, now I understand the thinking – thanks! I think this makes logical sense and is a good change regardless of whether we add internal state transitions.

The downside here is that it means increased latency – geography transitions would only be reported when the trip is complete. One of the goals of this PR is to allow cities to make realtime Policy decisions.

The other argument I'd make is that we are really just generalizing the existing pattern of firing events when municipal boundaries are crossed. As we know municipal boundaries can be quite complex and multi-layered. The way I think about is simple: _we communicate an event when a vehicle passes into an area where different rules apply_. That could be a Policy region under the same jurisdiction, a new jurisdiction – or potentially both.

Reply:
Spoke about this today, we've decided to go ahead with the proposal to include these new event types. There is some consideration of collapsing them into one event instead of two, that's a detail something we hope to discuss and resolve at tomorrow's WG.